releasenotes/notes/add-pauli-product-measurement-instruction-49abd4141328b46d.yaml

```
@@ -0,0 +1,29 @@
+---
+features_circuits:
+  - |
+    Added a new instruction class, :class:`.PauliProductMeasurement`, which represents
+    a fundamental operation in fault-tolerant quantum computing. 
+    Mathematically, it performs a joint projective measurement on
+    multiple qubits, where the measured observable is a tensor product of Pauli operators.
```

We can simplify this a bit (also in the class docstring):
```suggestion
    Added a new instruction class, :class:`.PauliProductMeasurement`, which represents
    a joint projective measurement on multiple qubits, where the measured observable is 
    a tensor product of Pauli operators.
```